src/main/java/appeng/client/render/cablebus/P2PTunnelFrequencyBakedModel.java

```diff
@@ -0,0 +1,142 @@
+
+package appeng.client.render.cablebus;
+
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import com.google.common.cache.Cache;
+import com.google.common.cache.CacheBuilder;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.renderer.block.model.BakedQuad;
+import net.minecraft.client.renderer.block.model.IBakedModel;
+import net.minecraft.client.renderer.block.model.ItemOverrideList;
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+import net.minecraft.client.renderer.vertex.VertexFormat;
+import net.minecraft.util.EnumFacing;
+
+import appeng.api.parts.IPartBakedModel;
+import appeng.api.util.AEColor;
+import appeng.util.Platform;
+
+
+public class P2PTunnelFrequencyBakedModel implements IBakedModel, IPartBakedModel
+{
+	private final VertexFormat format;
+	private final TextureAtlasSprite texture;
+
+	private final static Cache<Long, List<BakedQuad>> modelCache = CacheBuilder.newBuilder().maximumSize( 100 ).build();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)